tests/e2e/fastify-adapter.e2e-spec.ts

```diff
@@ -80,19 +80,6 @@ describe('Fastify adapter', () => {
     });
   });
 
-  describe('when trying to get a non-existing file', () => {
```

Comment:
this should remain the default behavior unless a flag (TBD which flag) is enabled. For express it is fallthrough: true so I guess we could use it for Fastify too

Reply:
You're right, I think it's a good idea, I just made a commit to re-use this flag on fastify, I also updated the tests, it seems to cover all the cases. Tell me what you think

Reply:
The description of this fallthrough option is misleading, maybe an `indexFallback` or `catchAll` option not in `serveStaticOptions` but in the root object can be more suited? It's only a matter of naming at this point. Tell me what you think about and I'll make the change